### PR TITLE
[LC] LC Scenes Integration Tests

### PR DESCRIPTION
**Pull request by lpbeliveau-silabs**
Added test for scenes integration to Level control cluster

Note:
An extra step was added to read the minimal level for level control, this will be added to the test plan once the current PR is approved.

### CODE REVIEW

**Review by jmartinez-silabs — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml [hunk 1]
```diff
@@ -0,0 +1,244 @@
+# Copyright (c) 2021 Project CHIP Authors
```
Comment (cecille) [inline src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml:1]:
```suggestion
# Copyright (c) 2024 Project CHIP Authors
```

File: src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+              - name: "Level"
+                value: 0
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 1
+              - name: "OptionsOverride"
+                value: 1
+
+    - label: "Step 2b: TH reads the MinLevel attribute from the DUT"
```
Comment (cecille) [inline src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml:141]: This doesn't match the test plan. 2b says:
TH reads OnOff attribute (On/Off cluster) from DUT

File: src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+                value: 0x00
+              - name: "GroupID"
+                value: G1
+              - name: "SceneList"
+                value: []
+
+    - label:
+          "Step 2a: TH sends a MoveToLevel command to DUT, with Level =0 and
+          TransitionTime =0 (immediate)"
+      cluster: "Level Control"
```
Comment (cecille) [inline src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml:128]: needs to be gated on the pics

File: src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+    - label: "Step 2b: TH reads the MinLevel attribute from the DUT"
+      cluster: "Level Control"
+      command: "readAttribute"
+      attribute: "MinLevel"
+      response:
+          saveAs: MinLevelValue
+          constraints:
+              type: int8u
+
+    - label: "Step 2c: TH reads the CurrentLevel attribute from DUT"
```
Comment (cecille) [inline src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml:150]: missing pics

File: src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+      cluster: "Level Control"
+      command: "readAttribute"
+      attribute: "CurrentLevel"
+      response:
+          value: MinLevelValue
+
```
Comment (cecille) [inline src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml:156]: missing steps 2d and 2e

File: src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+                value: G1
+              - name: "SceneID"
+                value: 0x01
+
+    - label: "Step 6c: TH reads the CurrentLevel attribute from DUT"
+      cluster: "Level Control"
+      command: "readAttribute"
+      attribute: "CurrentLevel"
+      response:
+          value: MinLevelValue
```
Comment (cecille) [inline src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml:244]: I know it's not part of the test plan, but you might want to consider adding a remove all scenes and remove all groups at the end so the device is left in the expected state.

File: src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml (hunk 5 above)
Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml:156]: discussed offline - test plan is being changed.

File: src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml (hunk 3 above)
Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml:128]: discussed on slack - test plan is being changed.

File: src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml (hunk 4 above)
Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml:150]: discussed on slack - test plan is being changed.

File: src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml (hunk 2 above)
Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml:141]: discussed on slack - test plan is being changed.

**Review by cecille — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml (hunk 2 above)
Comment (lpbeliveau-silabs) [reply src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml:141]: We are removing all cluster interaction between OnOff and Level Control in scenes recall, turns out it was not the expected behaviour, so every step/PICS related to OnOff has been removed. Will update Test Plan once the merges.

File: src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml (hunk 4 above)
Comment (lpbeliveau-silabs) [reply src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml:150]: Same as above regarding OnOff, and current level being a mandatory attribute, the LVL PICS became redundant.

File: src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml (hunk 3 above)
Comment (lpbeliveau-silabs) [reply src/app/tests/suites/certification/Test_TC_LVL_9_1.yaml:128]: Same as above regarding OnOff.

**Pull request merged**